fem/fespace.cpp

```diff
@@ -2800,7 +2800,8 @@ ElementRestriction::ElementRestriction(const FiniteElementSpace &f,
    {
       for (int d = 0; d < dof; ++d)
       {
-         const int gid = elementMap[dof*e + d];
+         const int gid = (elementMap[dof*e + d] >= 0) ? elementMap[dof*e + d] : -1 -
```

Comment:
I'd prefer to keep `gid` as is and work with a new variable to see better what's happening here.